pkg/apis/sriovnetwork/v1/sriovnetwork_types.go

```diff
@@ -16,6 +16,8 @@ type SriovNetworkSpec struct {
 	// +kubebuilder:validation:Minimum=0
 	// +kubebuilder:validation:Maximum=4096
 	Vlan             int    `json:"vlan,omitempty"`
+	Spoof            string `json:"spoof,omitempty"`
```

Comment:
@zshi-redhat to add any new field in CRD, you have to run `make gencode` to update the CRD template and deep-copy. You may also want to check https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md#add-a-new-custom-resource-definition. 

And there is issue of operator-sdk. So after running `make gencode`. Please make sure only the relevant files are updated, and the API group in CRD manifest is unchanged.

Reply:
Done